packages/engine/Source/Core/ITwinPlatform.js

```diff
@@ -0,0 +1,155 @@
+import Check from "./Check.js";
+import defined from "./defined.js";
+import DeveloperError from "./DeveloperError.js";
+import Resource from "./Resource.js";
+import RuntimeError from "./RuntimeError.js";
+
+/**
+ * Default settings for accessing the iTwin platform.
+ *
+ * @experimental This feature is not final and is subject to change without Cesium's standard deprecation policy.
+ *
+ * @see ITwinData for ways to import data
+ * @namespace ITwinPlatform
+ */
+const ITwinPlatform = {};
+
+/**
+ * Status states for a mesh-export export
+ * @enum {string}
+ */
+ITwinPlatform.ExportStatus = Object.freeze({
+  NotStarted: "NotStarted",
+  InProgress: "InProgress",
+  Complete: "Complete",
+  Invalid: "Invalid",
+});
+
+/**
+ * Types of mesh-export exports. CesiumJS only supports loading <code>3DTILES</code> type exports
+ * @enum {string}
+ */
+ITwinPlatform.ExportType = Object.freeze({
+  IMODEL: "IMODEL",
+  CESIUM: "CESIUM",
+  "3DTILES": "3DTILES",
+});
+
+/**
+ * Gets or sets the default iTwin access token. This token should have the <code>itwin-platform</code> scope.
+ *
+ * @experimental This feature is not final and is subject to change without Cesium's standard deprecation policy.
+ *
+ * @type {string|undefined}
+ */
+ITwinPlatform.defaultAccessToken = undefined;
+
+/**
+ * Gets or sets the default iTwin API endpoint.
+ *
+ * @experimental This feature is not final and is subject to change without Cesium's standard deprecation policy.
+ *
+ * @type {string|Resource}
+ * @default https://api.bentley.com
+ */
+ITwinPlatform.apiEndpoint = new Resource({
+  url: "https://api.bentley.com",
+});
+
+/**
+ * @typedef {Object} ExportRequest
+ * @property {string} iModelId
+ * @property {string} changesetId
+ * @property {ITwinPlatform.ExportType} exportType Type of the export. CesiumJS only supports the 3DTILES type
+ */
+
+/**
+ * @typedef {Object} Link
+ * @property {string} href
+ */
+
+/**
+ * @typedef {Object} ExportRepresentation
+ * The export objects from get-exports when using return=representation
+ * @property {string} id Export id
+ * @property {string} displayName Name of the iModel
+ * @property {ITwinPlatform.ExportStatus} status Status of this export
+ * @property {string} lastModified
+ * @property {ExportRequest} request Object containing info about the export itself
+ * @property {{mesh: Link}} _links Object containing relevant links. For Exports this includes the access url for the mesh itself
+ */
+
+/**
+ * @typedef {Object} GetExportsResponse
+ * @property {ExportRepresentation[]} exports The list of exports for the current page
+ * @property {{self: Link, next: Link | undefined, prev: Link | undefined}} _links Pagination links
+ */
+
+/**
+ * Get the list of exports for the specified iModel at it's most current version. This will only return exports with {@link ITwinPlatform.ExportType} of <code>3DTILES</code>.
```

Comment:
If I understand correctly, this is returning one export, not a list correct?

Reply:
Right now I'm only requesting a page of `$top=1` based on the assumption it should always exist with the auto-generation. That said this _is_ still returning a list matching the response from the API. It's just a list of length 1.